### DISCUSSION TITLE
FESOM Nodal Bottom Depth

### DISCUSSION THREAD

**Discussion by alxthor** [esm-tools/esm_tools#884] [Models]
I had a look at the core2 mesh files in order to perform vertical integration/averaging of output.
What is the difference between the values listed in aux3d.out (according to the [documentation](https://fesom2.readthedocs.io/en/latest/meshes/meshes.html) H_v is the depth at each vertex) and the values in the fesom.mesh.diag.nc (zbar_n_bottom is described as nodal bottom depth)?
The values in the mesh.diag seem to always match a FESOM level, but it is not always the closest level to the one in aux3d.out.

**Comment by mandresm** [comment]
Hi @alxthor, this question is about FESOM, so I recommend you to post it in the [FESOM GitHub](https://github.com/FESOM/fesom2/discussions) to ensure you get an answer :)
